### PR TITLE
Add client function examples

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
No description yet, you can take a look at the code.

**ClientFunction** from https://github.com/DevExpress/testcafe-examples/issues/2

### CODE REVIEW

File: examples/advanced/call-page-functions-with-arguments/call-page-functions-with-arguments.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+fixture `YouTube Playback`
+    .page `https://www.youtube.com/watch?v=I6WuCe1My9w`
+    .meta({ autoplay: true });
+
+test('YouTube Playback', async t => {
+    const play        = ClientFunction(() => document.querySelector('.video-stream').play());
+    const pause       = ClientFunction(() => document.querySelector('.video-stream').pause());
+    const currentTime = ClientFunction(() => document.querySelector('.video-stream').currentTime);
+    const setTime     = ClientFunction((time) => document.querySelector('.video-stream').currentTime = time);
```
Comment (miherlosev) [inline examples/advanced/call-page-functions-with-arguments/call-page-functions-with-arguments.js:11]: Redurant brackets, change to multiline expression:
```js
const setTime  = ClientFunction(time => {
   document.querySelector('.video-stream').currentTime = time;
};
```

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: examples/submit-a-form/submit-a-form.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { ClientFunction, Selector } from 'testcafe';
+
+fixture `Submit a Form`
+    .page `https://devexpress.github.io/testcafe/example`;
+
+test('Submit a Form', async t => {
+    const submitForm = ClientFunction(() => {
+        document.getElementById('main-form').requestSubmit();
```
Comment (miherlosev) [inline examples/submit-a-form/submit-a-form.js:8]: `document.getElementById('main-form').submit();`

File: examples/wait-for-iframe-load/wait-for-iframe-load.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+import { ClientFunction } from 'testcafe';
+
+const waitForIframeLoad = ClientFunction((iframeSelector) => new Promise((resolve, reject) => {
```
Comment (miherlosev) [inline examples/wait-for-iframe-load/wait-for-iframe-load.js:3]: `const waitForIframeLoad = ClientFunction(iframeSelector => ...`

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: examples/advanced/call-page-functions-with-arguments/call-page-functions-with-arguments.js [hunk 4]
```diff
@@ -0,0 +1,22 @@
+import { ClientFunction } from 'testcafe';
+
+fixture `YouTube Playback`
+    .page `https://www.youtube.com/watch?v=I6WuCe1My9w`
+    .meta({ autoplay: true });
```
Comment (miherlosev) [inline examples/advanced/call-page-functions-with-arguments/call-page-functions-with-arguments.js:5]: Why does the `meta` call is necessary (here and below)?

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: examples/submit-a-form/submit-a-form.js (hunk 2 above)
Comment (VasilyStrelyaev) [reply examples/submit-a-form/submit-a-form.js:8]: `form.submit()` doesn't trigger `onsubmit`, which is required on this page.
So, we have to do either `requestSubmit()` (not available in IE and Safari but we don't run these examples there) or `dispatchEvent(new Event('submit'))`

File: examples/advanced/call-page-functions-with-arguments/call-page-functions-with-arguments.js (hunk 4 above)
Comment (VasilyStrelyaev) [reply examples/advanced/call-page-functions-with-arguments/call-page-functions-with-arguments.js:5]: Looks nicer in the console IMHO:
https://github.com/DevExpress/testcafe-examples/pull/22/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R18

File: examples/submit-a-form/submit-a-form.js (hunk 2 above)
Comment (miherlosev) [reply examples/submit-a-form/submit-a-form.js:8]: ok

File: examples/wait-for-iframe-load/wait-for-iframe-load.js [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+        if (i > 60) {
+            window.clearInterval(intervalId);
+            reject(new Error('Iframe content loading timeout'))
+        }
+        i++;
+    }, 1000);
+}));
+
+fixture `Wait for an iframe to load`
+    .page `https://js.devexpress.com/Demos/WidgetsGallery/Demo/DataGrid/Overview/jQuery/Light/`;
```
Comment (miherlosev) [inline examples/wait-for-iframe-load/wait-for-iframe-load.js:25]: There is a simpler way:
```js
const iframeSelector = Selector(''#demoFrame', { timeout: 60000 });

await t.switchToIframe(iframeSelector);
```

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: examples/change-element-style/change-elements-style.js [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    });
+
+    await t.expect(populateButton.visible).ok();
+
+    await hidePopulateButton();
+
+    await t.expect(populateButton.visible).notOk();
+});
+
+test('Remove header color', async t => {
```
Comment (miherlosev) [inline examples/change-element-style/change-elements-style.js:19]: `test('Change header color'`

File: examples/change-element-style/README.md [hunk 7]
```diff
@@ -0,0 +1,27 @@
+# Change Element's Style
+
+**Test Code**: [change-elements-style.js](change-elements-style.js)
+
+This example shows how to modify an element's style from test code.
+
+The [change-elements-style.js](change-elements-style.js) file contains two tests. These tests use [client functions](https://devexpress.github.io/testcafe/documentation/guides/basic-guides/obtain-client-side-info.html) to run code in the browser.
+
+During the `Hide an element` test, the `hidePopulateButton` client function sets the `#populate` button's  [style.display](https://developer.mozilla.org/en-US/docs/Web/CSS/display) CSS property to `none`. Before the function is called, the [t.expect.ok](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) assertion verifies that the button is visible. After `hidePopulateButton` is executed, the [t.expect.notOk](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/notok.html) assertion ensures that the button is no longer displayed.
```
Comment (miherlosev) [inline examples/change-element-style/README.md:9]: `the [t.expect.ok](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) assertion **method** verifies`

here and below

File: examples/change-element-style/README.md [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
+
+In the `Remove header color` test, a client function modifies the `h1` header's [style.color](https://developer.mozilla.org/en-US/docs/Web/CSS/color) to change its color to black. The [t.expect.eql](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/expect/ok.html) assertions verify that the element's color has changed. To obtain the color value, the selector's [getStyleProperty](https://devexpress.github.io/testcafe/documentation/reference/test-api/domnodestate.html) method is used.
```
Comment (miherlosev) [inline examples/change-element-style/README.md:11]: In the `Change header color`

File: examples/wait-for-iframe-load/README.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
+# Wait For Iframe Loading
+
+**Test Code**: [wait-for-iframe-load.js](wait-for-iframe-load.js)
+
+TestCafe implements [wait mechanisms](https://devexpress.github.io/testcafe/documentation/guides/concepts/built-in-wait-mechanisms.html) that automatically suspend the test until all required page elements are loaded and ready for interaction. These mechanisms apply to page loads, animations, XHR requests, iframe initializations, etc. TestCafe waits until the target element is ready, or reports an error if this does not happen within a timeout.
```
Comment (miherlosev) [inline examples/wait-for-iframe-load/README.md:5]: This example demonstrates only the call `swithToIframe` method with a parameter.
It's better to add this example in the corresponding [help topic](https://devexpress.github.io/testcafe/documentation/reference/test-api/testcontroller/switchtoiframe.html).

**Review by miherlosev — CHANGES_REQUESTED** [review]

File: examples/call-client-functions-with-arguments/call-client-functions-with-arguments.js [hunk 10]
```diff
@@ -0,0 +1,14 @@
...
+fixture `Parameterized Client Functions`
+    .page('https://devexpress.github.io/testcafe/example/');
+
+const getLocationPart = ClientFunction(locationPart => {
+    return window.location[locationPart];
+});
+
+test('Parameterized Client Functions', async t => {
+    await t
+        .expect(getLocationPart('host')).eql('devexpress.github.io')
```
Comment (miherlosev) [inline examples/call-client-functions-with-arguments/call-client-functions-with-arguments.js:12]: I see it again and think that this example should be moved to the documentation.

Add `Client function with arguments` paragraph [here] (https://devexpress.github.io/testcafe/documentation/guides/basic-guides/obtain-client-side-info.html#execute-client-functions)

**Review by miherlosev — CHANGES_REQUESTED** [review]

**Comment by VasilyStrelyaev** [comment]
Examples transfered in https://github.com/DevExpress/testcafe/pull/5585

**Review by miherlosev — APPROVED** [review]

**Review by DIRECTcut — APPROVED** [review]

**Pull request merged**